api/files/client.go

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) 2020 SSH Communications Security Inc.
+//
+// All rights reserved.
+//
+
+package files
+
+import (
+	"github.com/SSHcom/privx-sdk-go/restapi"
+)
+
+// Client is a role-store client instance.
+type Client struct {
+	api restapi.Connector
+}
+
+// New creates a new client instance to fetch config files from PrivX
+func New(api restapi.Connector) *Client {
+	return &Client{api: api}
+}
+
+// ConfigExtender fetches configugartion file
+func (files Client) ConfigExtender(id string) ([]byte, error) {
+	var session struct {
```

Comment:
Shouldn't this be `func (client *Client) ConfigEntender(....)`?

Reply:
Sorry. I am not getting this. The component is called `extender` ?

Reply:
sorry for being unclear here... should this method be a Pointer Receiver? It does not alter `files` so it does not have to be, but having pointer receivers seems to be an established convention.